hil-test/tests/uart_async.rs

```diff
@@ -7,7 +7,7 @@
 //! Connect TX and RX pins.
 
 //% CHIPS: esp32 esp32c2 esp32c3 esp32c6 esp32h2 esp32s2 esp32s3
-//% FEATURES: generic-queue
```

Comment:
Was this intentional?

Reply:
Doing it, yes, leaving it in, no :D thanks